docs/crds/avinfrasetting.md

```diff
@@ -122,6 +122,13 @@ AviInfraSetting CRD can be used to enable/disable Route Health Injection (RHI) o
 
 This overrides the global `enableRHI` flag for the virtualservices corresponding to the AviInfraSetting.
 
+#### Enable/Disable Public IP
+
+AviInfraSetting CRD can be used to enable/disable Public IP on the virtualservices created by AKO. 
+
+        network:
+          enablePublicIP: true
```

Comment:
can you also add a statement that this is supported only on public clouds.

Reply:
updated as a note